docs/user/production-considerations/task-manager-production-considerations.asciidoc

```diff
@@ -0,0 +1,140 @@
+
+[float]
+[[task-manager-scaling-vertically]]
+==== Scaling Vertically
+
+Other times it, might be preferable to increase the throughput of individual {kib} instances.
+
+Tweak the *Max Workers* via the <<task-manager-settings,`xpack.task_manager.max_workers`>> setting, which would allow each {kib} to pull a higher number of tasks per interval, but keep in mind that this could impact the performance of each {kib} instance as _their_ workload would be higher.
+
+Tweak the *Poll Interval* via the <<task-manager-settings,`xpack.task_manager.poll_interval`>> setting, which would allow each {kib} to pull scheduled tasks at a higher rate, but keep in mind that this could impact the performance of the {es} cluster as _their_ workload would be higher.
+
+[float]
+[[task-manager-choosing-scaling-strategy]]
+==== Choosing a Scaling Strategy
+
+Each scaling strategy comes with its own considerations, and as a consequence, the appropriate strategy largely depends on your use case
+
+Scaling {kib} instances vertically causes higher resource utilization in each {kib} instance, as it will perform more concurrent work.
+Scaling {kib} instances horizontally requires a higher degree of coordination which can impact overall performance.
+
+Finding the sweet spot between both strategies requires experimentation, but in general our recommended strategy would be to follow these steps:
+
+1. Produce a <<task-manager-rough-throughput-estimation,rough throughput estimate>> as a guide to provisioning as many {kib} instances as needed. We recommend including any growth in tasks you predict experiencing in the near future, and a buffer to better address ad-hoc tasks.
+2. After provisioning a deployment as per the rough estimate, assess whether the provisioned {kib} instances achieve the required throughput by evaluating the <<task-manager-health-monitoring>> as described under <<task-manager-theory-insufficient-throughput,"Insufficient Throughtput to Handle the Scheduled Workload">>
+3. If the throughput is insufficient, and {kib} instances are exhibiting low resource utilization, incrementally scale vertically all the while <<kibana-page,monitoring>> the impact of these changes.
+4. If the throughput is insufficient, and {kib} instances are exhibiting high resource utilization, incrementally scale horizontally by provisioning new {kib} instances and reassess.
+
+{kib} Task Manager, like the rest of the Elastic Stack, has been designed to scale horizontally, and we recommend taking advantage of this ability to ensure mission ciritcal services such as Alerting and Reporting always have the capacity they need.
+
+Scaling horizontally requires a higher degree of coordination between {kib} instances. One way by which {kib} Task Manager coordinates with other instances is delaying its polling schedule to avoid conflicting with other instances.
```

Comment:
```suggestion
Scaling horizontally requires a higher degree of coordination between {kib} instances. One way Task Manager coordinates with other instances is by delaying its polling schedule to avoid conflicts with other instances.
```